Sources/MathParser/MathParser.swift

```diff
@@ -1,7 +1,8 @@
 // Copyright © 2021 Brad Howes. All rights reserved.
 
 import Parsing
-import XCTest
+import Foundation
+import XCTestDynamicOverlay
```

Comment:
I don't think this is necessary after swapping XCTest for Foundation.

Reply:
Without Foundation many standard library symbols are undefined: sin, cos, etc.

Reply:
I meant you are correct to do what you did but there is no reason to add the overlay import.

Reply:
I check out your project and everything will work just fine all of your changes, but you can remove the "import XCTestDynamicOverlay" -- it is not necessary. I can remove it after or just update the PR with a new commit.

Reply:
BTW, nice to meet you! You've helped me out a ton with your debugger.

![Capto_Capture 2023-04-19_08-55-23_PM](https://user-images.githubusercontent.com/686946/233173694-67328e4a-6485-4ad6-a5e0-b8c47764f817.png)